packages/components/src/components/step/step.stories.ts

```diff
@@ -21,207 +33,181 @@ export default {
+  name: 'As Link',
+  render: () => html`
+    <div class="w-min">
+      <sd-step
+        href="https://solid-design-system.fe.union-investment.de/x.x.x/storybook/?path=/docs/docs-general-introduction--docs"
+      >
+        <span slot="label">Step name</span>
+      </sd-step>
+    </div>
+  `
 };
 
 /**
- * Use the 'default', 'label', 'index', 'circle-content' slots to add content to the step..
+ * Use the `no-tail` attribute to remove the tail from a step.
  */
 
-export const Slots = {
-  parameters: {
-    controls: { exclude: ['default', 'label', 'index', 'circle-content'] }
-  },
-  render: () => {
-    return html`
-      ${['default', 'label', 'index', 'circle-content'].map(slot =>
-        generateTemplate({
-          axis: {
-            y: {
-              type: 'slot',
-              name: slot,
-              title: 'slot=..',
-              values: [
-                {
-                  value:
-                    slot === 'default'
-                      ? `<div class="slot slot--border slot--background h-12"></div>`
-                      : `<div slot='${slot}' class="slot slot--border slot--background h-12"></div>`,
-                  title: slot
-                }
-              ]
-            }
-          },
-          args: overrideArgs([
-            {
-              type: 'attribute',
-              name: 'current',
-              value: true
-            },
-            {
-              type: 'slot',
-              name: 'default',
-              value: `Lorem ipsum est dolor sit amet`
-            },
-            {
-              type: 'slot',
-              name: 'label',
-              value: `<span slot="label">Step name</span>`
-            }
-          ])
-        })
-      )}
-    `;
-  }
+export const NoTail = {
+  name: 'No Tail',
+  render: () => html`
+    <div class="w-min">
+      <sd-step no-tail>
+        <span slot="label">Step name</span>
+      </sd-step>
+    </div>
+  `
 };
 
 /**
- * sd-steps are fully accessibile via keyboard.
+ * Use the `not-interactive` attribute to make a step non-interactive.
  */
 
-export const Mouseless = {
-  render: (args: any) => {
-    return html`<div class="mouseless">${generateTemplate({ args })}</div>`;
-  },
-
-  play: async ({ canvasElement }: { canvasElement: HTMLUnknownElement }) => {
-    const el = canvasElement.querySelector('.mouseless sd-step');
-    await waitUntil(() => el?.shadowRoot?.querySelector('button'));
-    // We have to catch the event as otherwise Storybook will break
-    await userEvent.type(el!.shadowRoot!.querySelector('button')!, '{return}', { pointerEventsCheck: 0 });
-  }
+export const NotInteractive = {
+  name: 'Not Interactive',
+  render: () => html`
+    <div class="w-min">
+      <sd-step not-interactive>
+        <span slot="label">Step name</span>
+      </sd-step>
+    </div>
+  `
+};
+
+/**
+ * Use the `circle-content` slot to add a content-icon in a non-interactive step.
+ */
+
+export const Icon = {
+  name: 'Icon',
```

Comment:
Could you show step name pls

![image](https://github.com/user-attachments/assets/0c7aa788-c74e-4ff3-9728-9871dfedebe5)

Reply:
Updated